lib/rbs/cli.rb

```diff
@@ -808,8 +809,10 @@ def run_test(args, options)
         'RBS_TEST_TARGET' => (targets.join(',') unless targets.empty?)
       }
 
-      system(env_hash, *args)
-      $?
+      out, err, status = Open3.capture3(env_hash, *args)
+      stdout.print(out)
+      stderr.print(err)
+      status
```

Comment:
This is the only point to change the behavior of the implementation. Calling `system` doesn't respect `@stdout` and `@stderr` variables, so I changed to use `Open3.caputure3`. This change also leads to suppress outputs of `rbs test` while testing.